streams/src/main/java/org/apache/kafka/streams/processor/api/ProcessorContext.java

```diff
@@ -53,9 +53,9 @@
     TaskId taskId();
 
     /**
-     * The metadata of the source record, if is one. Processors may be invoked to
+     * Return the metadata of the current record if available. Processors may be invoked to
```

Comment:
```suggestion
     * Returns the metadata of the current record if available. Processors may be invoked to
```
An `s` is missing.

Reply:
We write JavaDocs imperatively. No `s` is intended.

Reply:
If've seen you have changed all the others to say `Return`, then this is fine